### PR TITLE
Move from nowcasting-infrastructure to ocf-infrastructure

### PR DESCRIPTION
**Pull request by devsjc**
# Pull Request

## Description
Since the SBRI work we are undertaking necessitates a new domain, this PR elects to change this `nowcasting` specific infrastructure repo, to instead contain code for all domains in an all-encompassing `ocf-infrastructure` repo. This was decided as a better alternative to having a new repo for infrastructure per-domain, as that obfuscates - or at least reduces the discoverability and ease of onboarding - for people new to the OCF github organisation.

Main changes:
  - Rename from `nowcasting-infrastructure` to `ocf-infrastructure`
  - Refactor current terraform code into `nowcasting` domain subfolder
  - Rewrite the `database` module to be more generic, into the new `postgres` module. Update all main.tf's to use this module.

## Requirements
Updates to workspaces in Terraform Cloud:

[Nowcasting Dev Workspace: ](https://app.terraform.io/app/openclimatefix/workspaces/nowcasting_infrastructure_development-eu-west-1) 
 - [x] Repo in Version Control settings from `openclimatefix/nowcasting_infrastructure` to `openclimatefix/ocf-infrastructure`
 - [ ] Working Directory from `terraform/development` to `terraform/nowcasting/development`
 
 [Nowcasting Prod Workspace: ](https://app.terraform.io/app/openclimatefix/workspaces/nowcasting_infrastructure_production-eu-west-1) 
 - [x] Repo in Version Control settings from `openclimatefix/nowcasting_infrastructure` to `openclimatefix/ocf-infrastructure`
 - [ ] Working Directory from `terraform/production` to `terraform/nowcasting/production`
 
 PVSite Workspace
 - [ ] Create new workspace for PVSite dev/prod

### CODE REVIEW

File: terraform/pvsite/development/main.tf [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+  region             = var.region
+  environment        = var.environment
+  db_subnet_group    = module.networking.private_subnet_group
+  vpc_id             = module.networking.vpc_id
+  db_name            = "pvsite"
+  rds_instance_class = "db.t3.small"
+}
+
+module "metrics" {
```
Comment (peterdudfield) [inline terraform/pvsite/development/main.tf:32]: remove

**Comment by peterdudfield** [comment]
Is it worth merging this into `development` first? The dev env works off that branch so we can check it first

File: terraform/pvsite/development/main.tf (hunk 1 above)
Comment (devsjc) [reply terraform/pvsite/development/main.tf]: Done 👍

**Comment by peterdudfield** [comment]
@all-contributors please add @devsjc  for code

**Pull request merged**